spec/dish_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'dish'
+
+describe Dish do
+	
+	let (:dish) {Dish.new("Water", 1)}
+
+	# need to learn how to write this test
+	it 'must be initialized with dish' 
+		#expect{ dish = Dish.new("Water", 1) }.
+	
+
```

Comment:
Don't need lines 7 - 9 as you are testing the initialize method in lines 12 to 19.